app.js

```diff
@@ -1,90 +1,156 @@
 let fs = require('fs'),
```

Comment:
One more important change you should make... it's worth naming all your test files differently. I don't think you should have the same filename in different places because it can be confusing.